docs/buildah-build.1.md

```diff
@@ -708,8 +708,9 @@ Skip stages in multi-stage builds which don't affect the target stage. (Default
 
 **--squash**
 
-Squash all of the image's new layers into a single new layer; any preexisting layers
-are not squashed.
+Squash all of the new image’s layers (including those inherited from a base image) into a single new layer.
+
+We squash images by default now. Try with --squash=false and --layer=false to just squash all of the image's new layers into a single new layer; any preexisting layers are not squashed.
```

Comment:
I don't like the We, should be buildah.  Also this sentence does not make sense.

--squash defaults to "false"  So we don't squash image by default?

Reply:
Buildah by default does not create layers and puts everything in a single layer unless `--layer=true` but keeps the base layer separately but using `--squash=true` ensures that all base layers and and new layers are squashed into one.

Reply:
When I read this I come away with the understanding that

from fedora
run touch /foobar

Would create a squashed image of fedora and /foobar in an image as opposed to  the fedora image plus a layer with the file /foobar in it.